builder/builder-next/builder.go

```diff
@@ -162,16 +163,23 @@ func (b *Builder) DiskUsage(ctx context.Context) ([]*types.BuildCache, error) {
 			Description: r.Description,
 			InUse:       r.InUse,
 			Shared:      r.Shared,
-			Size:        r.Size_,
-			CreatedAt:   r.CreatedAt,
-			LastUsedAt:  r.LastUsedAt,
-			UsageCount:  int(r.UsageCount),
+			Size:        r.Size,
+			CreatedAt: func() time.Time {
+				if r.CreatedAt != nil {
+					return r.CreatedAt.AsTime()
+				}
+				return time.Time{}
+			}(),
+			LastUsedAt: func() *time.Time { t := r.LastUsedAt.AsTime(); return &t }(),
```

Comment:
This is a bit hairy; also not sure why one was a pointer and the other one wasn't.

Reply:
Oh! I need to do a nil-check on `LastUsedAt` as well; not sure if it could ever be `nil`.

Reply:
Honestly not sure why some of these were pointers and some weren't from the original when removing gogo proto. I just kind of went with it.